db/migrate/20150202094307_add_range_to_trend_counters.rb

```diff
@@ -0,0 +1,8 @@
+class AddRangeToTrendCounters < ActiveRecord::Migration[6.0]
+  def change
+    add_column :trend_counters, :interval_start, :datetime
+    add_column :trend_counters, :interval_end, :datetime
+
+    Rake::Task['trends:reduce'].invoke
```

Comment:
We probably don't need this anymore - iirc it was used when we converted trends to intervals. The rake task could probably also be dropped

Reply:
Dropped from migration, unsure with the rake task :hot_face: